### DISCUSSION TITLE
Wildcard Certificate Error

### DISCUSSION THREAD

**Discussion by BigManDave** [sct/overseerr#2012] [Help]
Good Morning,
Raised this as a bug but got pointed here instead. Anyone have any thoughts?

Description

Can't configure the Plex connection to use https://plex.my.domain:443 using a wildcard certificate.
Hostname/IP does not match the certificate's altnames error

Version

1.25.0

Steps to Reproduce

Configure Plex with reverse proxy so that it is accessible on https://plex.my.domain:443
Proxy has a wildcard certificate
Attempt to configure Overseerr using manual Plex connection and the above settings
-Unable to connect or refresh list-
Switch configuration to http 32400 and refresh server list
-Can refresh list-
See below screenshot

Expected Behavior

Overseerr should accept the valid Plex connection

Screenshots

![image](https://user-images.githubusercontent.com/66840216/130420360-180bb966-a5e1-43b1-8eb3-b75df3a5042b.png)

Device

Desktop W10 Firefox

**Comment by danshilm** [comment]
You don't need to use one of the servers that's listed in the dropdown since you might not want to use one of the options returned. You can input the details manually and hit "Save Changes" instead. If Overseerr can't connect to your Plex server using those details, you will get an error message. If you get "Plex connection successfully established", you're all good.

**Comment by r-pufky** [comment]
Confirmed that this is still an issue. wildcard certificates will result in a plex connection failure with the message: 

```json
{
 "errorMessage": "Hostname/IP does not match certificate's altnames: IP: {REDACTED INTERNAL IP} is not in the cert's list: "
}
```

The only workaround I've found is to use the `plex.direct` DNS entry from plex.

Additionally it looks like there is some configuration that is not cleared when changing settings in the UI. Moving from the plex.direct DNS entry to a letsencrypt wildcard certificate (plex.example.com) produces the following error:

```json
{
 "errorMessage": "Hostname/IP does not match certificate's altnames: Host: plex.example.com. is not in the cert's altnames: DNS:*.{PLEX DIRECT HASH REDACTED}.plex.direct"
}
```
